randovania/games/dread/logic_database/Ghavoran.txt

```diff
@@ -1720,6 +1720,8 @@ Extra - asset_id: collision_camera_024
       Any of the following:
           After Ghavoran - Pulse Radar Enky and Speed Booster Conservation (Beginner) and Ballspark
           Bomb and Climb Sloped Tunnels (Expert)
+          # https://youtu.be/Q266PvBvip8
+          Slide and Cross Bomb Launch (Advanced) and Lay Cross Bomb
```

Comment:
A quirk of having the connection be `Pickup (Missile Tank) -> Door to Save Station East` is that the requirements of `Dock to Energy Recharge Station -> Pickup (Missile Tank)` also apply. Meaning for this Cross Bomb Launch you'd also need one of the following:
- Speed + Speed Conservation (Beginner) + Ice
- Cross Bomb Skip (Intermediate) + Spin Boost
- Movement (Beginner).

Should this connection be changed to `Dock to Energy Recharge Station -> Door to Save Station East`?

Reply:
Either that or change it back to trivial with Cross Bombs, which will satisfy the Cross Bomb Launch without any other tricks.

Reply:
That would solve things yeah. Should go with this.

Reply:
I figured I'd add this as Beginner, since it's in practice what happens when you try to make it all the way but don't quite make it.

Reply:
This works too. I like this solution more I think.